script/fs-bench/work/run.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+#   Copyright The containerd Authors.
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+set -euo pipefail
+
+# Set environemnt variable if you want use a customize fio image,
+# whose entrypoint must be the start of a fio test, and output all its logs
+# (stdio (in file `stdio`), bw_log, iops_log, and lat_log) to /output.
+# 256m_4t stands for 4 threads and each read 256MiB (1024MiB in total).
+IMAGE_LEGACY="${IMAGE_LEGACY:-docker.io/sequix/fio:legacy_256m_4t}"
+IMAGE_STARGZ="${IMAGE_STARGZ:-docker.io/sequix/fio:stargz_256m_4t}"
+IMAGE_ESTARGZ="${IMAGE_ESTARGZ:-docker.io/sequix/fio:estargz_256m_4t}"
```

Comment:
We should use `docker.io/stargz` organization here. I'll push these images to `docker.io/stargz` later.